source/compliance.txt

```diff
@@ -6,48 +6,189 @@ Compliance
+  <security-kms-encryption>`. 
+
+- Use :manual:`queryable encryption </core/queryable-encryption/>` to secure
+  queries on encrypted data. With queryable encryption, sensitive information
+  remains protected even when users run queries on data. 
+
+  Use well-researched non-deterministic encryption schemes to maintain
+  a balance between security and functionality.
+
+  With {+qe+}, you can perform the following tasks:
+
+  - Encrypt sensitive data fields from the client-side.
+  - Store sensitive data fields as fully randomized encrypted data on the database 
+    server-side.
+  - Run expressive queries on the encrypted data. Choose encryption configurations
+    that support expressive queries on the encrypted data.
+
+  MongoDB completes these tasks without the server having knowledge of the data
+  it's processing.
+
+  Sensitive data is encrypted throughout its lifecycle, in-transit, at-rest, in-use,
+  in logs, and backups. The data is only ever decrypted on the client-side,
+  since only you have access to the encryption keys.
+
+  You can set up {+qe+} using the following mechanisms:
+
+  - :manual:`Automatic Encryption <csfle-tutorial-automatic-encryption>`
+    enables you to perform encrypted read and write operations without
+    having to add explicit calls to encrypt and decrypt fields.
+  - :manual:`Manual Encryption <qe-tutorials-manual-encryption>` enables
+    you to perform encrypted read and write operations through your MongoDB
+    driver's encryption library. You must specify the logic for encryption
+    with this library throughout your application.
+
+.. _arch-center-compliance-data-sovereignty:
+
+Data Sovereignty
+~~~~~~~~~~~~~~~~
+
+{+service+} simplifies data sovereignty compliance through the following features:
+
+- Multi-cloud and multi-region {+clusters+}.
+
+  Use multi-region and multi-cloud {+clusters+} to distribute data across
+  geographically-diverse regions and cloud providers. To learn more,
+  see :atlas:`Create Multi-Cloud Multi-Region MongoDB Deployments for High
+  Availability and Workload Isolation <create-cluster-multi-region>`.
+
+  Create {+clusters+} in any of the 110+ cloud regions across the three
+  major cloud providers, |gcp|, |aws|, and |azure|. With {+service+},
+  you can deploy dedicated MongoDB {+clusters+} in a specific cloud
+  provider and region of your choice. This approach increases security,
+  reduces the risk of localized failures, and ensures data redundancy.
+
+  For example, a European company handling sensitive customer data can
+  choose to deploy their MongoDB {+cluster+} in a data center located
+  within the European Union, such as in Frankfurt, Germany, using |aws|
+  or another cloud provider. By doing so, the company ensures that all
+  data remains within EU borders and complies with the GDPR.
+
+- Global {+clusters+} with zoned sharding.
+
+  Use global {+clusters+} to control data storage locations. You can
```

Comment:
Reframe this as "you can" because this page isn't making recommendations (compliance recs will be in a future version)
```suggestion
  You can use global {+clusters+} to control data storage locations. You can
```